.docker/entrypoint/lib/commands/jekyll_builder.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'jekyll'
+
+# The Jekyll module contains everything related to Jekyll.
+module Jekyll
+  # The Jekyll::PlantUml module contains everything related to Jekyll::PlantUml.
+  module PlantUml
+    # The Jekyll::PlantUml::Commands module contains the implementations of the
+    # various commands that Jekyll PlantUML supports.
+    module Commands
+      # The Jekyll::PlantUml::JekyllBuilder class executes the Jekyll command
+      # 'build'.
+      class JekyllBuilder
+        def initialize(jekyll_config)
+          jekyll_config.must_be_a! Hash
+          raise ArgumentError, 'jekyll_config cannot be empty' if jekyll_config.empty?
```

Comment:
Codacy found an issue: [Modifier form of `if` makes the line too long.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5786435)